### ISSUE TITLE
Recursive ObjectType broken in monorepo since release 7.1.5 

### ISSUE THREAD

**Issue by aandrieu** [nestjs/graphql#756]
## I'm submitting a...

<pre><code>
[x] Regression 
[ ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

I have a monorepo setup, so it is based on webpack.

**Before v7.1.5 and in particular the commit** https://github.com/nestjs/graphql/commit/c0ddccd3b1d9808cb355d2ba598e088a3142cc6c

Two object type classes like that:
```typescript
// comment.model.ts
import { ObjectType, Field, Int } from '@nestjs/graphql';

@ObjectType()
export class Comment {
  @Field(type => Int)
  readonly id: number;

  readonly content: string;

  readonly children: Comment[];
}

// post.model.ts
import { ObjectType, Field, Int } from '@nestjs/graphql';

import { Comment } from '../comments/comment.model';

@ObjectType()
export class Post {
  @Field(type => Int)
  readonly id: number;

  readonly title: string;

  readonly content: string;

  readonly comments: Comment[];
}
```
was compiled like that:
```javascript
...
let Comment = class Comment {
    static _GRAPHQL_METADATA_FACTORY() {
        return { id: { nullable: false, type: () => Number }, content: { nullable: false, type: () => String },  , comment: { nullable: false, type: () => [__webpack_require__(16).Comment] } };
    }
};
...
let Post = class Post {
    static _GRAPHQL_METADATA_FACTORY() {
        return { id: { nullable: false, type: () => Number }, title: { nullable: false, type: () => String }, content: { nullable: false, type: () => String }, comments: { nullable: false, type: () => [__webpack_require__(16).Comment] } };
    }
};
...
```
So everything works well.

But since the patch, eager imports are added, and the comment class is missing for post:
```javascript
...
const eager_import_0 = __webpack_require__(17);
const eager_import_1 = __webpack_require__(18);
const eager_import_2 = __webpack_require__(16);
const graphql_1 = __webpack_require__(6);
let Comment = class Comment {
    static _GRAPHQL_METADATA_FACTORY() {
        return { id: { nullable: false, type: () => Number }, content: { nullable: false, type: () => String }, comment: { nullable: false, type: () => [__webpack_require__(16).Comment] } };
    }
};
...
const eager_import_0 = __webpack_require__(17);
const eager_import_1 = __webpack_require__(18);
const eager_import_2 = __webpack_require__(!(function webpackMissingModule() { var e = new Error("Cannot find module './comment.model'"); e.code = 'MODULE_NOT_FOUND'; throw e; }()));
const eager_import_3 = __webpack_require__(16);
const eager_import_4 = __webpack_require__(19);
const graphql_1 = __webpack_require__(6);
let Post = class Post {
    static _GRAPHQL_METADATA_FACTORY() {
        return { id: { nullable: false, type: () => Number }, title: { nullable: false, type: () => String }, content: { nullable: false, type: () => String }, comments: { nullable: false, type: () => [__webpack_require__(16).Comment] } };
    }
};
...
```

Sometime it will compile fine, but most of the time you get the error. I suppose it is related to the recursive field because as soon as you remove it, the error disappears.

## Expected behavior
Recursive object type classes should work as before.

## Minimal reproduction of the problem with instructions
https://github.com/aandrieu/nestjs-blog-test

1. npm i
2. nest start

you should see the error while nest start.

## What is the motivation / use case for changing the behavior?
It is a regression. Used to work well before this version.

## Environment

<pre><code>
Nest version: 7.0.7
Nest cli version: 7.1.2
Nest graphql version: 7.2.1
Graphql version: 15.0.0
 
For Tooling issues:
- Node version: 13.6.0
- Platform:  Mac
</code></pre>

**Comment by rained23** [comment]
I also encountered this bug, I updated my nest 7 using `nest update`, when I  removed `dist`, now my project is not able to be compiled, all my model and input will cause `Error: Cannot find module`

**Comment by lavaxun** [comment]
facing the same issue as well in 7.2.1

**Comment by kamilmysliwiec** [comment]
Fixed in 7.2.3. Please, let me know if you are still facing this issue (remember to remove `dist` folder)
